app/uk/gov/hmrc/tradergoodsprofiles/config/AppConfig.scala

```diff
@@ -31,7 +31,7 @@ class AppConfig @Inject() (config: Configuration, servicesConfig: ServicesConfig
   lazy val withdrawAdviceEnabled: Boolean = config.get[Boolean]("features.withdrawAdviceEnabled")
   lazy val requestAdviceEnabled: Boolean  = config.get[Boolean]("features.requestAdviceEnabled")
 
-  val isDrop1_1_enabled: Boolean =
+  lazy val isDrop1_1_enabled: Boolean =
     config
       .getOptional[Boolean]("features.drop_1_1_enabled")
       .getOrElse(false)
```

Comment:
@taylors-version  Do we need to include `features.drop_1_1_enabled` in config file for each env- like  here ? 
https://github.com/hmrc/app-config-development/blob/main/trader-goods-profiles.yaml

Reply:
It looks to me like the code is saying that if it's not in the config file, default to false. However it would be good to include it in the env configs for safety